WBooks/Book_Information/BookInformationViewController.swift

```diff
@@ -0,0 +1,122 @@
+        static let rentBookSuccessMessage = "Book rented"
+        static let rentBookFailureMessage = "The book could not be rented. Try again"
+    }
+    
+    // MARK: - Properties
+    
+    private lazy var _view: BookInformationView = BookInformationView.loadFromNib()!
+    private let _bookViewModel: BookInformationViewModel
+    
+    // MARK: - Initiaizers
+    
+    init(bookViewModel: BookInformationViewModel) {
+        _bookViewModel = bookViewModel
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        self.view = _view
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureDetails()
+        bindAddToWishlitButtonAction()
+        bindRentButtonAction()
+    }
+    
+    // MARK: - Private methods
+    
+    private func configureDetails() {
+        _view.titleLabel.text = _bookViewModel.title
+        _view.authorLabel.text = _bookViewModel.author
+        _view.releaseYearLabel.text = _bookViewModel.year
+        _view.genreLabel.text = _bookViewModel.genre
+        _view.bookCoverImageView.reactive.image <~ _bookViewModel.image
+        _bookViewModel.bookStatus.producer
+            .take(during: self.reactive.lifetime)
+            .startWithValues( {[unowned self] in
+            self._view.enableRentButton(isEnabled: $0.rentEnabled)
+            self._view.setStatusTo($0)
+        })
+    }
+    
+    private func bindAddToWishlitButtonAction() {
+        self._view.addToWishlistButton.reactive.controlEvents(.touchUpInside)
+            .take(during: self.reactive.lifetime)
+            .observeValues { [unowned self] _ in
+                self._bookViewModel.addBookToWishlit().startWithResult({ [unowned self] (result) in
+                    switch result {
+                    case .success:
+                        self.showMessage(Constants.addToWishlisSuccessMessage)
+                    case .failure:
+                        self.showMessage(Constants.addToWishlisFailureMessage)
+                    }
+                })
+        }
+    }
+    
+    private func bindRentButtonAction() {
+        self._view.rentButton.reactive.controlEvents(.touchUpInside)
+            .take(during: self.reactive.lifetime)
+            .observeValues { [unowned self] _ in
+                self._bookViewModel.rentBook().startWithResult({ [unowned self] (result) in
+                    switch result {
+                    case .success:
+                        self.showMessage(Constants.rentBookSuccessMessage)
+                    case .failure:
+                        self.showMessage(Constants.rentBookFailureMessage)
+                    }
+                })
+        }
+    }
+    
+    private func showMessage(_ text: String) {
```

Comment:
I love this ! It's sort of like a toast which iOS doesn't have natively. You could add it to Wolmo Core !